daprdocs/content/en/reference/components-reference/supported-conversation/openai.md

```diff
@@ -37,8 +41,10 @@ The above example uses secrets as plain strings. It is recommended to use a secr
 |--------------------|:--------:|---------|---------|
 | `key`   | Y | API key for OpenAI. | `mykey` |
 | `model` | N | The OpenAI LLM to use. Defaults to `gpt-4-turbo`.  | `gpt-4-turbo` |
-| `endpoint` | N | Custom API endpoint URL for OpenAI API-compatible services. If not specified, the default OpenAI API endpoint is used. | `https://api.openai.com/v1` |
+| `endpoint` | N | Custom API endpoint URL for OpenAI API-compatible services. If not specified, the default OpenAI API endpoint is used. Required when `apiType` is set to `azure`. | `https://api.openai.com/v1`, `https://example.openai.azure.com/` |
 | `cacheTTL` | N | A time-to-live value for a prompt cache to expire. Uses Golang duration format.  | `10m` |
+| `apiType` | N | Specifies the API provider type. Required when using a provider that does not follow the default OpenAI API endpoint conventions. | `azure` |
+| `apiVersion`| N | The API version to use. Required when the `apiType` is set to `azure`. | `2025-01-01-preview` |
```

Comment:
@fabistb  - Is `2025-01-01-preview` the only current valid value or are there others? If so, is there a link to Azure documentation for this list that we can reference?